cg/meta/workflow/microsalt/quality_controller/quality_controller.py

```diff
@@ -83,6 +84,9 @@ def quality_control_sample(self, sample_id: str, metrics: SampleMetrics) -> Samp
             and valid_10x_coverage
         )
 
+        if is_non_microsalt_apptag(sample):
```

Comment:
Is there a better name for such apptags?

Reply:
one solution is to create the opposite function, `has_sample_microsalt_apptag` and just check if that is False 😅

Reply:
you could name them "non_microbial" instead of "non_microsalt"